db/migrate/20160701161129_remove_non_alphanumeric_chars_from_developer_username.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveNonAlphanumericCharsFromDeveloperUsername < ActiveRecord::Migration
+  def up
+  	Developer.find_each do |developer|
```

Comment:
I think this method relies too much on the `Developer` model. There was a great blog post on this subject a while back; check it out: http://www.informit.com/articles/article.aspx?p=1652025

I'd suggest a solution like the one about halfway down the page-- defining the `Developer` model in the migration.